apps/flowlord/handler.go

```diff
@@ -407,9 +446,30 @@ func (tm *taskMaster) backload(req request) response {
 	}
 
 	for t := start; end.Sub(t) >= 0; t = byIter(t) {
-		tsk := *task.New(req.Task, tmpl.Parse(req.Template, t))
-		tsk.Meta = meta
-		tasks = append(tasks, tsk)
+		info := tmpl.Parse(req.Template, t)
+		tskMeta := make(url.Values)
+		tskMeta.Set("cron", t.Format(DateHour))
+		if workflowPath != "" {
+			tskMeta.Set("workflow", workflowPath)
+		}
+		if job := req.Job; job != "" {
+			tskMeta.Set("job", job)
+		}
+
+		for _, d := range data { // meta data tasks
+			tsk := *task.New(req.Task, tmpl.Meta(info, d))
+			for k, _ := range d {
+				tskMeta.Set(k, d.Get(k))
+			}
```

Comment:
I'm not sure if we should be setting the meta data for every value. The way this is written right now, If we have a json file with 100 fields, but only use 3 of those values all 100 fields would be added to the meta structure. 

1. We could leave this as it is
2. Only log the values that are used (change how we do meta templating
3. Don't add any of these values to the meta.

Reply:
I think things would break if we didn't add the meta for some of the fields, as we have processes that rely on those fields getting set.
I think if anything option 2 is the best way to go. though adding all of them ensures we have those values available. I don't think many would set a ton of values normally, so option 1 doesn't seem all that bad.

Reply:
I've implemented option number 2